database/entity/0053-change_password_encryption/UserContact.ts

```diff
@@ -0,0 +1,66 @@
+import {
```

Comment:
what is changed in this file?

Reply:
Why is this still in the changeset? Did I miss something? Why we need changes on UserContact?

Reply:
we need to keep the User joined with the UserContacts because of some path traversal references along the code